src/software/simulation/er_force_simulator_test.cpp

```diff
@@ -185,7 +185,8 @@ TEST_F(ErForceSimulatorTest, add_yellow_robots)
     EXPECT_TRUE(yellow_visible);
 }
 
-TEST_F(ErForceSimulatorTest, yellow_robot_velocity_test)
+// TODO why does this test fail?
```

Comment:
Seems to be a remaining TODO

Reply:
I think i need @jonathanlews  help on this one